.stylelintrc

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "stylelint-config-sass-guidelines",
+  "plugins": [
+    "stylelint-selector-bem-pattern"
+  ],
+  "rules": {
+    "max-nesting-depth": 4,
```

Comment:
4's a bit much— let's drop this to 2 and fix offending rules

Reply:
With BEM, 2 wouldn't leave room for any pseudo-class selectors on the Element. I'd recommend 3 at the most minimum.

Reply:
3 makes sense to me, though I don't think we should nest element selectors— ideally I think we'd only allow

- pseudo-class selectors
- modifiers
- modifiers + pseudo-class selectors